src/include/simeng/arch/aarch64/helpers/neon.hh

```diff
@@ -585,9 +590,14 @@ RegisterValue vecUMinP(srcValContainer& sourceValues) {
   const T* n = sourceValues[0].getAsVector<T>();
   const T* m = sourceValues[1].getAsVector<T>();
 
+  // Concatenate the vectors
+  T temp[2 * I];
+  memcpy(temp, m, sizeof(T) * I);
```

Comment:
Shouldn't `m` and `n` be switched here?

Reply:
Good spot, only updated maxP to be in-line with HW...